SofaKernel/modules/SofaDefaultType/src/sofa/defaulttype/typeinfo/models/IncompleteTypeInfo.h

```diff
@@ -27,7 +27,7 @@
 namespace sofa::defaulttype
 {
 template<class TDataType>
-struct SOFA_DEFAULTTYPE_API IncompleteTypeInfo
+struct IncompleteTypeInfo
```

Comment:
I'm not sure to understand why having the declspec is problematic in this case.

Reply:
Because the struct IncompleteTypeInfo is purely inline, so it does not need to export any symbols in the generated .lib

Reply:
As far as I know, inline is juste an hints for the compiler there is no guarantees that an inline will actually be inlined.Wrong I'm ?

Reply:
by pure inline, I wanted to say not present in a translation unit (aka a cpp).
In this case, you know the content of IncompleteTypeInfo (i.e the definitions) by including the header, not by linking a .lib
In MSVC you **must** say "I want to export the symbol(s)" when you compile a cpp. Here that not the case as there is not any symbol defined in a translation unit (cpp)

Reply:
👍